docs/response-issues.md

```
@@ -0,0 +1,832 @@
+   # POST /v1/responses/{id}/cancel during streaming
+   ```
+   
+   Expected:
+   - `responses.status` = `'cancelled'`
+   - `assistant_messages.status` = `'incomplete'`
+   - `assistant_messages.finish_reason` = `'cancelled'`
+
+## State Transitions
+
+### Before Fix
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)